File: Python/1-if-else/exercicio1.py
"""
Faça um programa que receba dois números e mostre qual deles é o maior.
"""

print('Digite o primeiro número: ')
numero_1 = input()

print('Digite o segundo número: ')
numero_2 = input()

if numero_1 > numero_2:
    print('Número 1 maior que número 2')
elif numero_1 == numero_2:
    print('Números iguais')
else:
    print('Número 2 maior que número 1')

File: Java/1-if-else/src/if_else/Exercicio1.java
package if_else;

import java.util.Scanner;

//Faça um programa que receba dois números e mostre qual deles é o maior.
public class Exercicio1 {
    public static void main(String[] args) {
        Scanner scan = new Scanner(System.in);

        int numero1;
        int numero2;

        System.out.println("Digite o primeiro número: ");
        numero1 = scan.nextInt();

        System.out.println("Digite o segundo número");
        numero2 = scan.nextInt();

        if(numero1 > numero2){
            System.out.println("Número 1 maior que número 2");
        } else if (numero1 == numero2){
            System.out.println("Números são iguais");
        } else {
            System.out.println("Número 2 maior que número 1");
        }
    }
}
